packages/labs/analyzer/src/lib/lit-element/events.ts

```diff
@@ -16,50 +16,37 @@ import {Event} from '../model.js';
 import {AnalyzerInterface} from '../model.js';
 import {getTypeForJSDocTag} from '../types.js';
 
-import {LitClassDeclaration} from './lit-element.js';
-
 /**
  * Returns an array of analyzer `Event` models for the given
  * ts.ClassDeclaration.
  */
-export const getEvents = (
-  node: LitClassDeclaration,
+export const addEventsToMap = (
+  tag: ts.JSDocTag,
+  events: Map<string, Event>,
   analyzer: AnalyzerInterface
 ) => {
-  const events = new Map<string, Event>();
-  const jsDocTags = ts.getJSDocTags(node);
-  if (jsDocTags !== undefined) {
-    for (const tag of jsDocTags) {
-      if (tag.tagName.text === 'fires') {
```

Comment:
Since we need to parse jsdoc tags for more than events, the tag iteration is moved into `lit-element.ts`, which calls out to `addEventsToMap`